### PR TITLE
Uses the correct jwks_uri for oidc configuration endpoint

### PR DESCRIPTION
**Pull request by marcosnils**
## Hey, I just made a Pull Request!

Fixed the jwks url for the oidc configuration endpoint 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4813-zpajp8qbqbhl5mcdmpzqp4k8ix3upero.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/604100d565f8171b6201f479

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4813-zpajp8qbqbhl5mcdmpzqp4k8ix3upero.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/604100d565f8171b6201f479

File: .changeset/violet-donuts-press.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+Change the jwks url value for the oidc configuration endpoint
```
Comment (Rugvip) [inline .changeset/violet-donuts-press.md:5]: Could prolly make this JWKS instead of add it to vocab tbh :p

**Review by Rugvip — APPROVED** [review]
Good catch, thanks! 👍

File: .changeset/violet-donuts-press.md (hunk 1 above)
Comment (marcosnils) [reply .changeset/violet-donuts-press.md:5]: SGTM. I'll change that

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4813-zpajp8qbqbhl5mcdmpzqp4k8ix3upero.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/604100d565f8171b6201f479

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**